docs/getting_started/demo_site.rst

```diff
@@ -0,0 +1,8 @@
+=========
+Demo site
+=========
+
+
+To create a new site on Wagtail we recommend the ``wagtail start`` command in :doc:`creating_your_project`, however a demo site exists containing example page types and models. The demo site is also our recommended basis for contributing to Wagtail itself.
```

Comment:
I don't understand the line "The demo site is also our recommended basis for contributing to Wagtail itself."